cairo/tests/ef_tests/helpers/load_state_tests.py

```diff
@@ -101,22 +115,9 @@ def add_block_to_chain(
     assert keccak256(rlp.encode(block.header)) == block_header_hash
     assert rlp.encode(block) == block_rlp
 
-    if not mock_pow:
-        load.fork.state_transition(chain, block)
-    else:
-        fork_module = importlib.import_module(
-            f"ethereum.{load.fork.fork_module}.fork"
-        )
-        with patch.object(
-            fork_module,
-            "validate_proof_of_work",
-            autospec=True,
-        ) as mocked_pow_validator:
-            load.fork.state_transition(chain, block)
-            mocked_pow_validator.assert_has_calls(
-                [call(block.header)],
-                any_order=False,
-            )
+    cairo_chain = cairo_run("state_transition", chain, block)
+    chain.blocks = cairo_chain.blocks
+    chain.state = cairo_chain.state
```

Comment:
chain == cairo_chain ?

Reply:
no, python frustrating stuff, basically if you modify an inner field of an object then because that object is passed by reference it's modified outside the funciton, but if you do chain = cairo_chain, then you bind a new object to `chain` and the original `chain` object passed to the function will not change